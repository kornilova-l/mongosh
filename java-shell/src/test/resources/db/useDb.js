// command
use admin
// command
db
