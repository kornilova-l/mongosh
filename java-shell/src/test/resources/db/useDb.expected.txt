switched to db admin
admin